package.json

```diff
@@ -76,6 +76,7 @@
     "jest": "^29.7.0",
     "jest-environment-jsdom": "^29.7.0",
     "jest-junit": "^15.0.0",
+    "jsdom": "^24.1.0",
```

Comment:
Do you think having this devDependency here would be unnecessary? As it is only used on your app so would it be good to specify it in the package.json of your app?

Reply:
This week I was encountering the issue that not having that devDependency and specific version caused my app to be successful in my local environment but then fail on the pipeline when running `pnpm test`